README.md

```diff
@@ -114,28 +116,28 @@ Alternatively, you can also use a pre-built bundle from Transloadit's CDN: Edgly
 - ⓒ [`Google Drive`](https://uppy.io/docs/google-drive/)
 - ⓒ [`Dropbox`](https://uppy.io/docs/dropbox/)
 - ⓒ [`Instagram`](https://uppy.io/docs/instagram/)
-- ⓒ [`Import From URL`](https://uppy.io/docs/url/) — support picking files from remote providers or direct URLs from anywhere on the web
+- ⓒ [`Import From URL`](https://uppy.io/docs/url/) — supports picking files from remote providers or direct URLs from anywhere on the web
```

Comment:
“support” was meant like a call to action here, as in “support picking files from anywhere on your website”.

Reply:
I see this now, thank you for clarifying.